### ISSUE TITLE
Certificate error when connecting to remote minio instance

### ISSUE THREAD

**Issue by phucduong86** [minio/mcs#189]
Hi, 

I am trying to use mcs with my existing minio instance.
I've created the mcs account as instructed with admin access and ran the following command to spin up the console server.

```
docker run -p 9090:9090 -e MCS_MINIO_SERVER=https://<server> \
-e MCS_ACCESS_KEY=mcs -e MCS_SECRET_KEY=mcs -e MCS_HMAC_JWT_SECRET=YOURJWTSIGNINGSECRET \
-e MCS_PBKDF_PASSPHRASE=SECRET -e MCS_PBKDF_SALT=SECRET minio/mcs:latest server --host 0.0.0.0 --port 9090
```

Once the console is up (http://localhost:9090) I try to login.
It takes up to 30secs or so and then return an error:

```
2020/07/03 19:16:00 Serving mcs at http://[::]:9090
2020/07/03 19:18:28 error obtaining MinIO region: Get https://<server>/minio/admin/v3/help-config-kv?key=&subSys=region: x509: certificate signed by unknown authority
```

Not sure how would I approach this issue,.
Thanks

**Comment by dvaldivia** [comment]
I believe we have a hiccup in the docker release file and we are not copying the standard CA certificates, will send a PR for this soon. @Alevsk do you agree with this probable cause?

**Comment by Alevsk** [comment]
Hi @phucduong86 if your minio instance is running with TLS enabled and it's using a self-signed certificate or a certificate issued by a custom certificate authority you can pass the certificate using the `MCS_MINIO_SERVER_TLS_ROOT_CAS` env variable

**Comment by phucduong86** [comment]
Thank you for the responses @Alevsk @dvaldivia 
I will give it a try.

**Comment by phlbrz** [comment]
Hello all,

Same problem here. 

Image: `minio/mcs:v0.1.1` logs:

```
2020/07/08 05:43:26 Serving mcs at http://[::]:9090
--
  | 2020/07/08 05:46:29 error obtaining MinIO region: Get https://minio-internal-service.myproject.svc.cluster.local:9000/minio/admin/v3/help-config-kv?key=&subSys=region:  x509: certificate signed by unknown authority
```

Image: minio/minio:RELEASE.2020-07-02T00-15-09Z

```
API: SYSTEM()
--
  | Time: 05:46:00 UTC 07/08/2020
  | DeploymentID: 791804f2-1ab0-4108-b97a-45b59ebad298
  | Error: http: TLS handshake error from 10.3.82.1:37158: remote error: tls: bad certificate
  | 1: github.com/minio/minio@/cmd/server-main.go:459:cmd.serverMain.func2()
```

I followed https://github.com/minio/minio-operator installation, as it says that "Automatic TLS for MinIO" at https://github.com/minio/minio-operator/blob/master/docs/tls.md#automatic-csr-generation with `requestAutoCert: true`.
I thought it should approve my cert and, as it says, handle with my pods.
Does it need to do another configuration or is it just a workaround for now?

- Some configuration:

I'm using Openshift 3.11, I had to make change my `minio-operator.yaml` to deploy, I changed `subresources` to be after `names` and before `versions`.

```
  names:
    kind: MinIOInstance
    singular: minioinstance
    plural: minioinstances
  subresources:
    # status enables the status subresource.
    status: {}
  versions:
....
```

Thanks.

**Comment by phlbrz** [comment]
Hello, 

Same problem with Image: `minio/mcs:v0.2.0`.

Thank's!

**Comment by dvaldivia** [comment]
this is due to the missing root CAs will put them in the container

**Comment by phlbrz** [comment]
> MCS_MINIO_SERVER_TLS_ROOT_CAS

Using it as workaround solved.

But with operator I don't have an option to add it in values before oc apply.
It could have it as an option.

I followed the instructions in https://github.com/minio/minio-operator#create-a-minio-instance
Downloaded and modified https://github.com/minio/minio-operator/blob/master/examples/minioinstance-kes.yaml due to my environment and updated some images.

```
  mcs:
    image: minio/mcs:v0.2.0
    replicas: 2
    mcsSecret:
      name: mcs-secret
    metadata:
      labels:
        app: mcs

```

I think if has some environments and mount in the yml should help.
I used minio-tls (auto-generated by operator) with mcs to solve.
Mounted a volume from this secret:

```
volume-cvayo
Type:
secret (populated by a secret when the pod is created)
Secret:
minio-tls
Key to File:
public.crt → certs/public.crt
```

And added the environment in my deploymentconfig:

```
MCS_MINIO_SERVER_TLS_ROOT_CAS: /tmp/certs/public.crt
```
